### DISCUSSION TITLE
I cant downgrade.

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#706] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
when i try to downgrade a chromebook i get "you are using an out-of-date chrome OS recovery image" and i cant get past it, can anyone help?

im using stable-channel octopus

**Comment by 3kh0** [comment, marked as answer]
yeah it has been patched, sorry

**Reply by ghost** [reply]
oh okay, thanks for the answer
